README.md

```diff
@@ -3,64 +3,78 @@
 
-## Setup Guide
-This is the step-by-step Inspetor integration:
-
-### Library Repositories
+## How to use
 The Inspetor iOS Library can be installed through [CocoaPods](https://cocoapod.org). To install all you have to do is follow this steps:
 
-1) Add this to your Podfile: `pod 'Inspetor'` (this uses the latest version).
-2) Run `pod install`.
+1. Add our library to you Podfile (`pod Inspetor`)
+1. Run `pod install` command
+
+P.S: When you import the Inspetor library you will see that you are actually installing more than one library, since the Inspetor iOS Library have some depndencies.
 
-When you import the Inspetor library you will see that you are actually installing more than one library, since the Inspetor iOS Library have some depndencies.
+## API Docs
+You can find more in-depth documentation about our frontend libraries and integrations in general [here](https://inspetor.github.io/docs-frontend).
 
 ### Library setup
-You must provide your config to setup our library. To do that, you need to pass us your configurations through our `setup()` function. All the access to the Inspetor functions is made through our `sharedInstance()` that is available by calling the `Inspetor.sharedInstance()`. There is an example in the code snipet bellow.
+In order to properly relay information to Inspetor's processing pipeline, you'll need to provide customer-specific authentication credentials:
+- App ID (provided by Inspetor)
+- Tracker name (provided by Inspetor)
 
-You **should** instantiate the Inspetor Library in your application `AppDelegate` in the `didFinishLaunchingWithOptions` function.
+With these, you can instantiate the Inspetor tracking instance. Our integration library instantiates a singleton instance to prevent multiple trackers from being instantiated, which could otherwise result in duplicate or inconsistent data being relayed to inspetor. Apart from that, the singleton will let you configure the library only once.
+
+The singleton instance is instantiated as follows:
 
 ```
 do {
-  try Inspetor.sharedInstance().setup(appId: "123", trackerName: "cool.name", devEnv: true)
+  try Inspetor.sharedInstance().setup(appId: "appId", trackerName: "trackerName", devEnv: false)
 } catch TrackerException.requiredConfig(let code, let message) {
   print("code: \(code) - message: \(message)")
 } catch {
   print("Error")
 }
 ```
 
-The ***"appId"*** is an unique identifier that the Inspetor Team will provide to you. The ***"trackerName"*** is a name that will help us to find your data in our database and we'll provide you with a couple of them. The ***"devEnv"*** is a boolean statement that you set to indicate that your are testing the library (development enviroment). It's set to false by default.
+Be advised, that this function can throw an exception if you pass invalids (empty strings or not in the format required) appId or/and trackerName.
```

Comment:
I don't think there should be a comma after `Be advised`. Also, I'd write the last bit of it as `if you pass an invalid appId and/or trackerName (empty strings or not in the format required).`